packages/website/content/pages/about.json

```diff
@@ -27,8 +27,8 @@
               "push_left": "off-6_sm-5_mi-1_ti-0"
             },
             "format": "header",
-            "heading": "What is Web3.Storage?",
-            "subheading": "Web3.Storage is a service to make building on top of Filecoin as simple as possible - giving developers the power of distributed storage and content addressing via simple client libraries or an HTTP API."
+            "heading": "What is web3.storage?",
+            "subheading": "web3.storage is a service to make building on top of Filecoin as simple as possible - giving developers the power of distributed storage and content addressing via simple client libraries or an HTTP API."
```

Comment:
I noticed that this change is different than the other proposed change over here https://github.com/web3-storage/web3.storage/pull/1942/files#diff-f437afa6570f3cd6d0b90a033364954e6898e8e17ae6cd177501d7451dc148faR31

It seems like only one open PR should be changing this? or that they should change it to the same thing

Reply:
I think we wanted the change from the other PR here.  I'm reverting this commit.  Sorry about all the confusion.